src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEquals.cs

```diff
@@ -144,25 +142,25 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
                 retExpr = F.ObjectNotEqual(other, F.Null(F.SpecialType(SpecialType.System_Object)));
             }
 
-            var recordProperties = ArrayBuilder<FieldSymbol>.GetInstance();
+            var fields = ArrayBuilder<FieldSymbol>.GetInstance();
             foreach (var member in ContainingType.GetMembers())
```

Comment:
>GetMembers [](start = 50, length = 10)

This does not include fieldlike event backing fields.

Reply:
Good catch. Updated and test added.